src/views/EditorHome.vue

```diff
@@ -778,6 +779,23 @@ const loadDraggedFile = (event: { dataTransfer: DataTransfer | null }) => {
       });
   }
 };
+
+// AudioCellの自動スクロール
+const cellsRef = ref<InstanceType<typeof draggable> | undefined>();
+watch(activeAudioKey, (audioKey) => {
+  if (audioKey == undefined) return;
+  const activeCellElement = audioCellRefs[audioKey].$el;
+  const cellsElement = cellsRef.value?.$el;
+  if (activeCellElement instanceof Element && cellsElement instanceof Element) {
+    const activeCellRect = activeCellElement.getBoundingClientRect();
+    const cellsRect = cellsElement.getBoundingClientRect();
+    const overflowTop = activeCellRect.top <= cellsRect.top;
+    const overflowBottom = activeCellRect.bottom >= cellsRect.bottom;
+    if (overflowTop || overflowBottom) {
+      activeCellElement.scrollIntoView(overflowTop || !overflowBottom);
+    }
+  }
+});
```

Comment:
仮に`cellsRef`がundefinedになっていたとき、ここのif内が実行されず、かつエラーも出ないのでバグに気づけないかもです。
throwにしちゃおうと思います！

```suggestion
  if (
    !(activeCellElement instanceof Element) ||
    !(cellsElement instanceof Element)
  )
    throw new Error(
      `invalid element: activeCellElement=${activeCellElement}, cellsElement=${cellsElement}`
    );
  const activeCellRect = activeCellElement.getBoundingClientRect();
  const cellsRect = cellsElement.getBoundingClientRect();
  const overflowTop = activeCellRect.top <= cellsRect.top;
  const overflowBottom = activeCellRect.bottom >= cellsRect.bottom;
  if (overflowTop || overflowBottom) {
    activeCellElement.scrollIntoView(overflowTop || !overflowBottom);
  }
});
```